administrator/language/en-GB/en-GB.com_joomlaupdate.ini

```
@@ -73,6 +76,12 @@ COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSION_WARNING_UNKNOWN="Unkown error"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSION_SERVER_ERROR="Update Server error"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS="Extensions Pre-Update Check"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_NONE="No extensions installed."
+COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_PLUGIN_BEING_CHECKED="The system is currently checking these plugins to see if they could cause problems during the upgrade.<br>Please be patient while the checks are completed."
```

```suggestion
COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_PLUGIN_BEING_CHECKED="Checking these plugins to see if they could cause problems during the upgrade.<br>Please be patient while the checks are completed."
```
Not really sure when this is displayed as I couldnt see it BUT need to be change the language or it might be interpreted as saying that joomla is checking the code in the plugins

---

If you switch to the 'live update' tab as soon as you load the update page then its shown next to the list of plugins.  

It only really shows up if the update checks are being slow and is mean to encourage patience!  'The system is currently checking' was meant to convey that the website is doing something right now and the user needs to wait.